Backend/routes/auth.js

```diff
@@ -1,6 +1,9 @@
 const express = require("express");
 const router = express.Router();
 const mongoose = require("mongoose");
+const jwt = require("jsonwebtoken");
+const secretKey = "your-secret-key"; // replace with your own secret key
```

Comment:
```suggestion
const secretKey = process.env.SECRET_KEY
```

Reply:
it's ok for testing right now, these can abstracted when configuring for deployment